src/Bridges/NetteDI/OrmExtension.php

```diff
@@ -31,22 +37,23 @@ class OrmExtension extends CompilerExtension
 	/** @var string */
 	protected $modelClass;
 
-	/** @var array<mixed> */
-	private $configDefaults = [
-		'model' => Model::class,
-		'repositoryFinder' => PhpDocRepositoryFinder::class,
-		'initializeMetadata' => false,
-	];
 
+	public function getConfigSchema(): Schema
+	{
+		return Expect::structure([
+			'model' => Expect::string()->default(Model::class),
+			'repositoryFinder' => Expect::string()->default(PhpDocRepositoryFinder::class),
+			'initializeMetadata' => Expect::string()->default(false),
```

Comment:
`Expect::bool()` for the `initializeMetadata` maybe?

Reply:
Yeah, that's correct 🤦‍♂️. I'll write some tests once I find out how to run them locally
https://github.com/nextras/orm/pull/609